IdentityCore/src/cache/ios/MSIDKeychainTokenCache.m

```diff
@@ -357,6 +357,8 @@ - (BOOL)removeItemsWithKey:(MSIDTokenCacheKey *)key
             *error = MSIDCreateError(NSOSStatusErrorDomain, status, @"Failed to remove items from keychain.", nil, nil, nil, context.correlationId, nil);
         }
         MSID_LOG_ERROR(context, @"Failed to delete keychain items (status: %d)", (int)status);
+        
+        return NO;
```

Comment:
`SecItemDelete` could return `errSecItemNotFound`, do we want to return YES in such a case?

Reply:
good point, fixed :)